examples/main_refactor.py

```diff
@@ -0,0 +1,165 @@
+"""
```

Comment:
This file could be removed, and its content directly replace the original file main_vep.py.
This started as a clone of main_vep.py